queries/highlights.scm

```diff
@@ -0,0 +1,33 @@
+(doctype) @constant
```

Comment:
Thanks for adding these in, that should unblock this grammar from being integrated with GitHub. It looks like tree-sitter-elixir has [unit tests for syntax highlighting](https://tree-sitter.github.io/tree-sitter/syntax-highlighting#unit-testing) that should probably be added here as well. What do you think?

Reply:
I think the highlight tests might have some trouble with languages where comments don't extend to end of line: I was getting some wonky results trying to test tree-sitter-eex. I'll give it another try and see if I can figure out why the tests are acting strangely :+1: